### PR TITLE
Changed product names in the documentation

### PR DESCRIPTION
**Pull request by davidrichards-da**
CHANGELOG_BEGIN
CHANGELOG_END

Terminology changes made:
- Removing reference to 'Connect'
- Changing the product names

### CODE REVIEW

File: docs/source/support/overview.rst [hunk 1]
```diff
@@ -37,14 +37,14 @@ At the bottom of every Daml Application is a Daml network, a distributed, or pos
...
 
 Ledger API
 **********
 
-The Ledger API is the primary interface that offers forward and backward compatibility between Daml Networks and Applications (including Daml Connect components). As you can see in the diagram above, all interaction between components above the Participant Node and the Participant Node or Daml Network happen through the Ledger API. The Ledger API is a public API and offers the lowest level of access to Daml Ledgers supported for application use.
+The Ledger API is the primary interface that offers forward and backward compatibility between Daml Networks and Applications (including Daml components). As you can see in the diagram above, all interaction between components above the Participant Node and the Participant Node or Daml Network happen through the Ledger API. The Ledger API is a public API and offers the lowest level of access to Daml Ledgers supported for application use.
 
-Daml Connect
+Daml Components
 ************
```
Comment (cocreature) [inline docs/source/support/overview.rst:48]:
```suggestion
Daml Components
***************
```

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -38,10 +38,10 @@ To install the SDK on Mac or Linux open a terminal and run:
 The installer will setup the ``PATH`` variable for you. In order for it to take effect, you will have to
 log out and log in again.
 
-Installing the Enterprise Edition
+Installing Daml Enterprise
 *********************************
```
Comment (stefanobaghino-da) [inline docs/source/getting-started/installation.rst:42]:
```suggestion
Installing Daml Enterprise
**************************
```
I believe the linter will complain about the title and the `***` not having the same size.

File: docs/source/ops/connect/auth0.rst [hunk 3]
```diff
@@ -593,17 +593,16 @@ in the folder that contains both ``nginx`` and ``my-project``:
...
-our frontend. All we need now is to get a Daml Connect system up and running.
+our frontend. All we need now is to get a Daml system up and running.
 We document two paths forward here: one that relies on the Helm chart included
-in Daml Connect Enterprise Edition, and a manual setup using only the Community
-Edition SDK.
+in Daml Enterprise, and a manual setup using only the Open Source SDK.
 
-Using the Connect Helm Chart
+Using the Daml Helm Chart
 ****************************
```
Comment (stefanobaghino-da) [inline docs/source/ops/connect/auth0.rst:601]:
```suggestion
Using the Daml Helm Chart
*************************
```
Same as above.

File: docs/source/ops/connect/auth0.rst [hunk 4]
```diff
@@ -673,7 +672,7 @@ domain on which your server is exposed. And voilà! Your application is up and
 running. You should be able to log in with Auth0, exchange messages, and set up
 an auto-reply trigger, all by connecting your browser to ``https://$DOMAIN/``.
 
-Manually Setting Up the Connect Components
+Manually Setting Up the Daml Components
 ******************************************
```
Comment (stefanobaghino-da) [inline docs/source/ops/connect/auth0.rst:676]:
```suggestion
Manually Setting Up the Daml Components
***************************************
```

File: docs/source/ops/connect/helm.rst [hunk 5]
```diff
@@ -3,17 +3,17 @@
 
 .. _connect-helm-chart:
 
-Connect Helm Chart
+Helm Chart
 ==================
```
Comment (stefanobaghino-da) [inline docs/source/ops/connect/helm.rst:7]:
```suggestion
Daml Helm Chart
===============
```
The point is always avoiding linter errors but I also added `Daml` back to be consistent with the edit you made before.

File: docs/source/ops/connect/helm.rst [hunk 6]
```diff
@@ -244,15 +244,15 @@ those databases as a whole, just like you would any other database.
...
 
 If you are running the Helm chart solely for the HTTP JSON API Service
 (connected to an external Ledger API server), then you can eschew backing up
 entirely, as the database for the HTTP JSON API Service is an
 easy-to-reconstruct cache. This assume that, in this setup, the data store of
 the Ledger API server is, itself, properly backed up.
 
-Securing Daml Connect
+Securing Daml
 ---------------------
```
Comment (stefanobaghino-da) [inline docs/source/ops/connect/helm.rst:256]:
```suggestion
Securing Daml
-------------
```
You know the drill.

File: docs/source/ops/connect/index.rst [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
 .. _ops-connect_index:
 
-Operating Daml Connect
+Operating Daml
 ======================
```
Comment (stefanobaghino-da) [inline docs/source/ops/connect/index.rst:7]:
```suggestion
Operating Daml
==============
```

File: docs/source/support/compatibility.rst [hunk 8]
```diff
@@ -14,7 +14,7 @@ Network Upgradeability
 
   Ledger Operators should be able to upgrade Daml network or Participant Nodes seamlessly to stay up to date with the latest features and fixes. A Daml application should be able to operate without significant change across such Network Upgrades.
 
-Daml Connect Upgradeability
+Daml Upgradeability
```
Comment (stefanobaghino-da) [inline docs/source/support/compatibility.rst:17]: Should there be the markup for a header here?

File: docs/source/support/compatibility.rst [hunk 9]
```diff
@@ -33,16 +33,16 @@ Participant Nodes advertise the Ledger API version they support via the :ref:`ve
 
 As a concrete example, Daml for Postgres 1.4.0 has the Participant Node integrated, and exposes Ledger API version 1.4.0 and the Daml for VMware Blockchain 1.0 Participant Nodes expose Ledger API version 1.6.0. So any application that runs on Daml for Postgres 1.4.0 will also run on Daml for VMware Blockchain 1.0.
 
-List of Ledger API Versions supported by Daml Connect
+List of Ledger API Versions supported by Daml
 =====================================================
```
Comment (stefanobaghino-da) [inline docs/source/support/compatibility.rst:37]:
```suggestion
List of Ledger API Versions supported by Daml
=============================================
```

File: docs/source/support/compatibility.rst [hunk 10]
```diff
@@ -86,22 +86,22 @@ Given the Ledger API Compatibility above, network upgrades are seamless if they
 
 As an example, from an application standpoint, the only effect of upgrading Daml for Postgres 1.4.0 to Daml for Postgres 1.6.0 is an uptick in the Ledger API version. There may be significant changes to components or database schemas, but these are not public APIs. 
 
-SDK, Runtime Component, and Library Compatibility: Daml Connect Upgradeability
+SDK, Runtime Component, and Library Compatibility: Daml Upgradeability
 ******************************************************************************
```
Comment (stefanobaghino-da) [inline docs/source/support/compatibility.rst:90]:
```suggestion
SDK, Runtime Component, and Library Compatibility: Daml Upgradeability
**********************************************************************
```

File: docs/source/tools/trigger-service/auth0_example.rst [hunk 11]
```diff
@@ -142,10 +142,10 @@ Enter the `details page <auth0-user-details_>`_ of the newly created user.
 .. _auth0-create-user: https://auth0.com/docs/users/create-users
 .. _auth0-user-details: https://auth0.com/docs/users/view-user-details
 
-Start Daml Connect
+Start Daml
 ~~~~~~~~~~~~~~~~~~
```
Comment (stefanobaghino-da) [inline docs/source/tools/trigger-service/auth0_example.rst:146]:
```suggestion
Start Daml
~~~~~~~~~~
```

**Review by stefanobaghino-da — COMMENTED** [review]
The suggestions _should_ make the build pass.

File: docs/source/support/compatibility.rst (hunk 8 above)
Comment (davidrichards-da) [reply docs/source/support/compatibility.rst:17]: I don't think so, it looks intentional on [the page](https://docs.daml.com/support/compatibility.html).  There are three terms in that section, none of which have headers.

**Pull request merged**